mim/commands/download.py

```diff
@@ -47,17 +54,20 @@ def cli(package: str,
         > mim download mmcls --config resnet18_8xb16_cifar10
         > mim download mmcls --config resnet18_8xb16_cifar10 --dest .
     """
-    download(package, configs, dest_root)
+    download(package, configs, check_certificate, dest_root)
 
 
 def download(package: str,
              configs: List[str],
+             check_certificate: bool = True,
              dest_root: Optional[str] = None) -> List[str]:
     """Download checkpoints from url and parse configs from package.
 
     Args:
         package (str): Name of package.
         configs (List[str]): List of config ids.
+        check_certificate (bool): Whether to check the ssl certificate.
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.